src/appState.ts

```diff
@@ -39,6 +44,7 @@ export const getDefaultAppState = (): Omit<
     elementType: "selection",
     errorMessage: null,
     exportBackground: true,
+    exportScale: defaultExportScale,
```

Comment:
I'm not sure about this one since it makes `getDefaultAppState` potentially non-deterministic. Granted, we're using `getDateTime()` for `title`, but I believe we wanted to get rid of it at some point.

Reply:
Thinking about it, I suppose I could have a default scale of null, and a getter that fills in the devicePixelRatio on demand. Would that be more reasonable?

Reply:
I'd use `1` as default value as `null` isn't valid, and `1` feels reasonable.

Reply:
Thanks for doing the merge conflicts for me.

I have made exportScale default to 1 as you suggest. Hopefully this has worked properly (I did it in the github text editor on my dad's Surface :-D).

This will be a behaviour change on most new-ish apple devices, because they all report themselves as high-dpi, and would normally end up with a default export scale of 2. Hopefully this is fine?

Reply:
I thought up till now we've always used `1` for scale by default?

Reply:
It's completely device dependant. On my phone it is even 3x by default. Should I regret those last few commits?

Reply:
Ok, I dunno how I missed that, sorry! Yes, let's keep the behavior then and for now let's add that value to `getDefaultAppState()` as you did initially. We can figure out later whether we need to factor it out elsewhere, or not.

Reply:
I have no idea how to revert or force push commits out of existence in a web browser. This might have to wait until tomorrow.

Reply:
No prob. Wanted to do it myself but my git somehow refuses to fetch/checkout your branch right now :man_shrugging:

Reply:
\o/ I managed to install git in termux on my phone, and did it in there.

Reply:
yay